portfolio/src/main/webapp/script.js

```diff
@@ -28,11 +28,39 @@ function addRandomQuoteOrFact() {
 }
 
 function hello(){
-   fetch('/data').then(response=>response.json()).then((hi)=>{
-    document.getElementById('response').innerText = hi;
-  });
+   fetch('/data').then(response=>response.json()).then((comment)=>{
+    const commentList = document.getElementById('response');
+    comment.forEach((com) => {commentList.appendChild(createListElement(com));
+    });
+    console.log(comment);
+    
+});
 }
 
+    function requestTranslation() {
+        const text = document.getElementById('comment').value;
+        console.log(text);
```

Comment:
also delete these console.log() debugging codes after you confirm everything is working right